target/scripts/helper-functions.sh

```diff
@@ -15,6 +15,18 @@ function escape
   echo "${1//./\\.}"
 }
 
+function print_environment
+{
+  _notify 'inf' 'BEGIN ENVIRONMENT'
+
+  for VARIABLE in "${!VARS[@]}"
+  do
+    echo "${VARIABLE}=${VARS[${VARIABLE}]}"
+  done
+
+  _notify 'inf' 'END ENVIRONMENT\n'
+}
+
```

Comment:
Do we have any chance to mask sensitive values by default?

Reply:
That would be a nice addition, but would require a lot of additional work on our side that is not worth it IMHO.

Reply:
Then maybe add an hint to remove sensitive values in this output :)

Reply:
Done. [ba9ea5a85f812e811782bb27864d87ed2916c056]